README

```diff
@@ -6,3 +6,13 @@ non-root users to view older versions of files and revert changes.
 
 For more information visit http://snapper.io/.
 
+Releasing
+=========
+
+Before releasing package ensure that changes made to package is mentioned
+in `package/snapper.changes`.
+To create package use commands `make -f Makefile.repo && make package`.
+Then use common work-flow to send package to build service. For factory
```

Comment:
To submit the package into Factory /maybe with link/, submit it, at first, into [YaST:Head](https://build.opensuse.org/project/show/YaST:Head) project in OBS.